mm2src/mm2_main/src/lp_swap.rs

```diff
@@ -677,15 +707,20 @@ pub fn active_swaps_using_coins(ctx: &MmArc, coins: &HashSet<String>) -> Result<
     Ok(uuids)
 }
 
-pub fn active_swaps(ctx: &MmArc) -> Result<Vec<Uuid>, String> {
+pub fn active_swaps(ctx: &MmArc) -> Result<Vec<(Uuid, u8)>, String> {
     let swap_ctx = try_s!(SwapsContext::from_ctx(ctx));
     let swaps = try_s!(swap_ctx.running_swaps.lock());
```

Comment:
We should panic (rather than returning an error) if mutex is poisoned (I know you haven't changed that; I thought it's worth to change it in this PR).